akka-actor-typed/src/main/mima-filters/2.6.1.backwards.excludes/issue-28161-logger-name.excludes

```diff
@@ -0,0 +1 @@
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.typed.internal.ActorContextImpl#LoggingContext.this")
```

Comment:
not sure if mima plugin handles files without new lines at the end?

Reply:
according to contributing.md the directory name should be 2.6.0...
`<last-version>.backwards.excludes`